repo/migrations/Migration009.go

```diff
@@ -0,0 +1,386 @@
+	RatingCount   uint32                 `json:"ratingCount"`
+	ModeratorIDs  []string               `json:"moderators"`
+}
+
+type Migration009_listingDataAfterMigration struct {
+	Hash          string                 `json:"hash"`
+	Slug          string                 `json:"slug"`
+	Title         string                 `json:"title"`
+	Categories    []string               `json:"categories"`
+	NSFW          bool                   `json:"nsfw"`
+	CoinType      string                 `json:"coinType"`
+	ContractType  string                 `json:"contractType"`
+	Description   string                 `json:"description"`
+	Thumbnail     Migration009_thumbnail `json:"thumbnail"`
+	Price         Migration009_price     `json:"price"`
+	ShipsTo       []string               `json:"shipsTo"`
+	FreeShipping  []string               `json:"freeShipping"`
+	Language      string                 `json:"language"`
+	AverageRating float32                `json:"averageRating"`
+	RatingCount   uint32                 `json:"ratingCount"`
+	ModeratorIDs  []string               `json:"moderators"`
+
+	// Adding AcceptedCurrencies
+	AcceptedCurrencies []string `json:"acceptedCurrencies"`
+}
+
+type Migration009_listing struct {
+	Listing Migration009_listing_listing `json:"listing"`
+}
+type Migration009_listing_listing struct {
+	Metadata Migration009_listing_listing_metadata `json:"metadata"`
+}
+type Migration009_listing_listing_metadata struct {
+	AcceptedCurrencies []string `json:"acceptedCurrencies`
+}
+
+var (
+	Migration009CreateCasesTable     = "create table cases (caseID text primary key not null, buyerContract blob, vendorContract blob, buyerValidationErrors blob, vendorValidationErrors blob, buyerPayoutAddress text, vendorPayoutAddress text, buyerOutpoints blob, vendorOutpoints blob, state integer, read integer, timestamp integer, buyerOpened integer, claim text, disputeResolution blob, lastDisputeExpiryNotifiedAt integer not null default 0);"
+	Migration009CreateSalesTable     = "create table sales (orderID text primary key not null, contract blob, state integer, read integer, timestamp integer, total integer, thumbnail text, buyerID text, buyerHandle text, title text, shippingName text, shippingAddress text, paymentAddr text, funded integer, transactions blob, needsSync integer, lastDisputeTimeoutNotifiedAt integer not null default 0);"
+	Migration009CreateSalesIndex     = "create index index_sales on sales (paymentAddr, timestamp);"
+	Migration009CreatePurchasesTable = "create table purchases (orderID text primary key not null, contract blob, state integer, read integer, timestamp integer, total integer, thumbnail text, vendorID text, vendorHandle text, title text, shippingName text, shippingAddress text, paymentAddr text, funded integer, transactions blob, lastDisputeTimeoutNotifiedAt integer not null default 0, lastDisputeExpiryNotifiedAt integer not null default 0, disputedAt integer not null default 0);"
```

Comment:
I like to indicate in the variable name that these schemas are as they existed prior to this migration.